crates/core/src/factories/mem_peer_store/test.rs

```diff
@@ -51,7 +51,7 @@ impl AgentBuild {
         });
         let is_tombstone = self.is_tombstone.unwrap_or(false);
         let url = self.url.unwrap_or(None);
-        let storage_arc = self.storage_arc.unwrap_or(BASIC_ARC_FULL);
+        let storage_arc = self.storage_arc.unwrap_or(BasicArc::Full);
```

Comment:
Per my previous comment about getting rid of the Full variant, we could add a const to the impl:

```rust
impl BasicArc {
    pub const FULL: Self = Self::Arc(0, 0);
}
```

So that this would become `BasicArc::FULL`.

Reply:
Done